sdk/cognitivelanguage/azure-ai-language-conversations/setup.py

```diff
@@ -1,39 +1,91 @@
+            'Uninstall it with "pip uninstall azure".'
+        )
+    except AttributeError:
+        pass
+except ImportError:
+    pass
 
-# To install the library, run the following
-#
-# python setup.py install
-#
-# prerequisite: setuptools
-# http://pypi.python.org/pypi/setuptools
+# Version extraction inspired from 'requests'
+with open(os.path.join(package_folder_path, '_version.py'), 'r') as fd:
+    version = re.search(r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]',
+                        fd.read(), re.MULTILINE).group(1)
 
-REQUIRES = ["msrest>=0.6.21", "azure-core<2.0.0,>=1.16.0"]
+if not version:
+    raise RuntimeError('Cannot find version information')
+
+with open('README.md', encoding='utf-8') as f:
+    readme = f.read()
+with open('CHANGELOG.md', encoding='utf-8') as f:
+    changelog = f.read()
 
 setup(
-    name=NAME,
-    version=VERSION,
-    description="azure-ai-language-conversations",
-    author_email="",
-    url="",
-    keywords=["Swagger", "ConversationAnalysisClient"],
-    install_requires=REQUIRES,
-    packages=find_packages(),
+    name=PACKAGE_NAME,
+    version=version,
     include_package_data=True,
-    long_description="""\
-    This API accepts a request and mediates among multiple language projects, such as LUIS Generally Available, Question Answering, LUIS Deepstack, and then calls the best candidate service to handle the request. At last, it returns a response with the candidate service's response as a payload.
-
- In some cases, this API needs to forward requests and responses between the caller and an upstream service.
-    """
-)
+    description='Microsoft {} Client Library for Python'.format(PACKAGE_PPRINT_NAME),
+    long_description=readme + "\n\n" + changelog,
+    long_description_content_type='text/markdown',
+    license='MIT License',
+    author='Microsoft Corporation',
+    author_email='azpysdkhelp@microsoft.com',
+    url='https://github.com/Azure/azure-sdk-for-python',
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'License :: OSI Approved :: MIT License',
+    ],
+    zip_safe=False,
+    packages=find_packages(exclude=[
+        'tests',
+        # Exclude packages that will be covered by PEP420 or nspkg
+        'azure',
+        'azure.ai',
+    ]),
+    install_requires=[
+        "azure-core<2.0.0,>=1.19.0",
+        "msrest>=0.6.21",
+        'azure-common~=1.1',
+        'six>=1.11.0',
+    ],
+    extras_require={
+        ":python_version<'3.0'": ['azure-ai-nspkg'],
+        ":python_version<'3.5'": ['typing'],
+    }
```

Comment:
add

    project_urls={
        'Bug Reports': 'https://github.com/Azure/azure-sdk-for-python/issues',
        'Source': 'https://github.com/Azure/azure-sdk-python',
    }